Tools/DB/db.go

```diff
@@ -0,0 +1,63 @@
+package db
+
+// All configuration for connection on database
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/msosic97/korpa-backend/config"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+
+
+
+var DB *sql.DB
+
+//  Connection to DB
+func StartDB(config *config.Config) (*sql.DB, error) {
+
+// Validation
+	if len(config.DBName) == 0 {
```

Comment:
this can be in separate function Validate and then call it here

Reply:
to write separate Validate function only for DBName or for all other fields?